exchange_info_service.go

```diff
@@ -45,5 +45,16 @@ type Symbol struct {
 	QuotePrecision     int                 `json:"quotePrecision"`
 	OrderTypes         []string            `json:"orderTypes"`
 	IcebergAllowed     bool                `json:"icebergAllowed"`
-	Filters            []map[string]string `json:"filters"`
```

Comment:
it is better to change `[]map[string]string` to `[]map[string]interface{}`, since the filters are likely to be changed in the future. btw, you can run gofmt as in https://travis-ci.org/adshao/go-binance/builds/408201495?utm_source=github_status&utm_medium=notification before submitting pr.

Reply:
@adshao done. Thanks for the feedback!